<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>18</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleLabel" name="TitleLabel">
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SmoothScrollArea" name="adv_scroll">
     <property name="styleSheet">
      <string notr="true">background: transparent; border: none</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>627</width>
        <height>694</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_21">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="SubtitleLabel" name="SubtitleLabel_3">
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_7">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_7">
                <property name="text">
                 <string>您的服务器地址(包括 http:// 或 https://)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_6">
                <property name="text">
                 <string>输入您的服务器地址</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="lightColor" stdset="0">
                 <color alpha="150">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="darkColor" stdset="0">
                 <color alpha="200">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="LineEdit" name="server">
              <property name="maximumSize">
               <size>
                <width>275</width>
                <height>33</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>例如：127.0.0.1:443</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_9">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_10">
                <property name="text">
                 <string>密钥（Secret）</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_9">
                <property name="text">
                 <string>输入您设定的密钥以正常上传状态</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="lightColor" stdset="0">
                 <color alpha="150">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="darkColor" stdset="0">
                 <color alpha="200">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="PasswordLineEdit" name="secret">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>225</width>
                <height>33</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>您的服务器密钥</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_7">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_7">
                <property name="text">
                 <string>代理</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_6">
                <property name="text">
                 <string>输入您的代理服务器信息</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="lightColor" stdset="0">
                 <color alpha="150">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="darkColor" stdset="0">
                 <color alpha="200">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="ComboBox" name="protocol"/>
            </item>
            <item>
             <widget class="LineEdit" name="proxy_server">
              <property name="maximumSize">
               <size>
                <width>275</width>
                <height>33</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>代理服务器:端口</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="LineEdit" name="user">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>275</width>
                <height>33</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>代理用户名(可选)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="PasswordLineEdit" name="pass">
              <property name="maximumSize">
               <size>
                <width>275</width>
                <height>33</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>代理密码(可选)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="SubtitleLabel" name="SubtitleLabel_4">
           <property name="text">
            <string>设备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_11">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_9">
                <property name="text">
                 <string>设备 ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_8">
                <property name="text">
                 <string>填写您的设备 ID</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="lightColor" stdset="0">
                 <color alpha="150">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="darkColor" stdset="0">
                 <color alpha="200">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="LineEdit" name="device_id">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>33</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>例如：pc-1</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="SubtitleLabel" name="SubtitleLabel_5">
           <property name="text">
            <string>关于</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_12">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_15">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_11">
                <property name="text">
                 <string>Sleepy Client</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_10">
                <property name="text">
                 <string>Copyright © RinLit 2025. All Rights Reserved.</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="lightColor" stdset="0">
                 <color alpha="150">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="darkColor" stdset="0">
                 <color alpha="200">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="HyperlinkLabel" name="github_link">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>GitHub</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="HyperlinkLabel" name="bilibili_link">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>哔哩哔哩</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="blank">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HyperlinkLabel</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PasswordLineEdit</class>
   <extends>LineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
